### PR TITLE
Container post action check

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Adding in check for container being ready in the pod once it gets killed

### CODE REVIEW

File: kraken/pod_scenarios/setup.py [hunk 1]
```diff
@@ -59,18 +59,13 @@ def container_run(kubeconfig_path, scenarios_list, config, failed_post_scenarios
...
                 # capture start time
                 start_time = int(time.time())
-                container_killing_in_pod(cont_scenario)
+                killed_containers = container_killing_in_pod(cont_scenario)
                 logging.info("Waiting for the specified duration: %s" % (wait_duration))
                 time.sleep(wait_duration)
-                failed_post_scenarios = post_actions.check_recovery(
-                    kubeconfig_path, container_scenario_config, failed_post_scenarios, pre_action_output
-                )
+                failed_post_scenarios = check_failed_containers(killed_containers, cont_scenario.get("retry_wait", 120))
```
Comment (chaitanyaenr) [inline kraken/pod_scenarios/setup.py:67]: Maybe move this check to line 65? This way the scenario waits for the container to recover and then waits for the usual time specified in the config. Thoughts?

Comment (paigerube14) [reply kraken/pod_scenarios/setup.py:67]: In pod scenarios we do the wait duration first and then do the check post actions. I could see it being helpful both ways. I could change either to match. definitely think we should be consistent

File: docs/container_scenarios.md [hunk 2]
```diff
@@ -14,4 +14,25 @@ scenarios:
...
+-   container_scenarios:                                 # List of chaos pod scenarios to load
+            - -    scenarios/container_etcd.yml
+              -    scenarios/post_action_etcd_container.py
+```
+
+2. Allow kraken to wait and check the killed containers become ready again. Kraken keeps a list of the specific
+containers that were killed as well as the namespaces and pods to verify all containers that were affected recover properly
+
+```
+retry_wait: <seconds to wait for container to recover
```
Comment (chaitanyaenr) [inline docs/container_scenarios.md:37]: Missing > in the end.

**Review by chaitanyaenr — APPROVED** [review]
Minor nit, other than that LGTM.

File: docs/container_scenarios.md (hunk 2 above)
Comment (paigerube14) [reply docs/container_scenarios.md]: Added!

**Pull request merged**